src/app/private/scripts/controllers/dashboard/orgDashboardController.js

```diff
@@ -60,10 +69,11 @@ angular.module('playerApp')
                   })
                   data.push(dataArray)
 
+                  var name = ''
```

Comment:
This can optimised  without else by assigning else value on **name** initialization